README.md

```diff
@@ -22,6 +22,9 @@ Official binary sources (provided by davidgfnet):
 Building
 --------
 
+Dependencies:
+Libc6-dev, libfreeimage-dev
```

Comment:
Should be something like: "Development headers and libraries for **glibc** and **Freeimage**. You'll also need a C/C++ compiler and linker such as GCC".
Possibly add a debian example: "Example (debian): sudo apt-get install build-essential libfreeimage-dev" [untested, not a debian user myself but I do realize that especially debain users need help with this..]